environments/development/common/cloudfront-policy.tf

```
@@ -55,6 +55,22 @@ resource "aws_cloudfront_response_headers_policy" "this" {
       protection = false
     }
   }
+
+  cors_config {
+    access_control_allow_origins  = {
+      items = ["*"]
+    }
```

```suggestion
    access_control_allow_origins {
      items = ["*"]
    }
```

These should be blocks, I think